src/evo/evodb.h

```diff
@@ -98,6 +98,7 @@ class CEvoDB
 
     bool CommitRootTransaction();
 
+    bool HasSomeBestBlock() { return db.Exists(EVODB_BEST_BLOCK); }
```

Comment:
```suggestion
    bool HasBestBlock() { return db.Exists(EVODB_BEST_BLOCK); }
```

nit, im good either way but feels like it would fit better with the `Some` dropped?

Reply:
That's what I did initially :) However `HasSomething()` functions _usually_ expect an actual value of `Something` to compare to and in this case we are fine with whatever record we have.  Would `HasAnyBestBlock()` be better? Or maybe smth even more general like `HasRecords()`?

Reply:
Ahh got your point. True, most time there is a parameter required but are at least two others, `HasWallets()` and `HasSelected()` without parameter. `IsEmpty()` which is also used a lot generally would probably also work. I don't have a strong opinion here in either direction.. Im good with all currently proposed/existing solutions really, let me know what you pick :D

Reply:
`IsEmpty()` is a great idea, should have thought of it from the start 🙈 fixed :)